2024-10-03 18:05:18,698 INFO    MainThread:3252 [wandb_setup.py:_flush():68] Configure stats pid to 3252
2024-10-03 18:05:18,698 INFO    MainThread:3252 [wandb_setup.py:_flush():68] Loading settings from C:\Users\shanthod\.config\wandb\settings
2024-10-03 18:05:18,698 INFO    MainThread:3252 [wandb_setup.py:_flush():68] Loading settings from C:\Users\shanthod\OneDrive - Singapore Management University\Desktop\codes\wandb\settings
2024-10-03 18:05:18,698 INFO    MainThread:3252 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2024-10-03 18:05:18,698 INFO    MainThread:3252 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program_relpath': 'main.py', 'program': 'C:\\Users\\shanthod\\OneDrive - Singapore Management University\\Desktop\\codes\\main.py'}
2024-10-03 18:05:18,698 INFO    MainThread:3252 [wandb_init.py:_log_setup():492] Logging user logs to C:\Users\shanthod\OneDrive - Singapore Management University\Desktop\codes\wandb\run-20241003_180518-cdvaxk9z\logs\debug.log
2024-10-03 18:05:18,698 INFO    MainThread:3252 [wandb_init.py:_log_setup():493] Logging internal logs to C:\Users\shanthod\OneDrive - Singapore Management University\Desktop\codes\wandb\run-20241003_180518-cdvaxk9z\logs\debug-internal.log
2024-10-03 18:05:18,698 INFO    MainThread:3252 [wandb_init.py:_jupyter_setup():438] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000226D161CDF0>
2024-10-03 18:05:18,702 INFO    MainThread:3252 [wandb_init.py:init():532] calling init triggers
2024-10-03 18:05:18,702 INFO    MainThread:3252 [wandb_init.py:init():538] wandb.init called with sweep_config: {}
config: {}
2024-10-03 18:05:18,702 INFO    MainThread:3252 [wandb_init.py:init():554] re-initializing run, found existing run on stack: 7fsiidh9
2024-10-03 18:05:18,704 INFO    MainThread:3252 [wandb_run.py:_finish():1854] finishing run shan_proj/ssl-relation-prediction/7fsiidh9
2024-10-03 18:05:18,704 INFO    MainThread:3252 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2024-10-03 18:05:18,704 INFO    MainThread:3252 [wandb_run.py:_restore():2066] restore
2024-10-03 18:05:18,705 INFO    MainThread:3252 [wandb_run.py:_restore():2072] restore done
2024-10-03 18:05:23,718 INFO    MainThread:3252 [wandb_run.py:_footer_history_summary_info():3442] rendering history
2024-10-03 18:05:23,718 INFO    MainThread:3252 [wandb_run.py:_footer_history_summary_info():3474] rendering summary
2024-10-03 18:05:23,730 INFO    MainThread:3252 [wandb_run.py:_footer_sync_info():3398] logging synced files
2024-10-03 18:05:23,740 INFO    MainThread:3252 [wandb_init.py:init():588] starting backend
2024-10-03 18:05:23,740 INFO    MainThread:3252 [wandb_init.py:init():592] setting up manager
2024-10-03 18:05:23,744 INFO    MainThread:3252 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn, using: spawn
2024-10-03 18:05:23,747 INFO    MainThread:3252 [wandb_init.py:init():599] backend started and connected
2024-10-03 18:05:23,759 INFO    MainThread:3252 [wandb_run.py:_label_probe_notebook():1203] probe notebook
2024-10-03 18:05:23,759 INFO    MainThread:3252 [wandb_run.py:_label_probe_notebook():1213] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-10-03 18:05:23,759 INFO    MainThread:3252 [wandb_init.py:init():687] updated telemetry
2024-10-03 18:05:23,769 INFO    MainThread:3252 [wandb_init.py:init():727] communicating run to backend with 60.0 second timeout
2024-10-03 18:05:25,349 INFO    MainThread:3252 [wandb_run.py:_on_init():2134] communicating current version
2024-10-03 18:05:25,413 INFO    MainThread:3252 [wandb_run.py:_on_init():2143] got version response upgrade_message: "wandb version 0.18.3 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-10-03 18:05:25,413 INFO    MainThread:3252 [wandb_init.py:init():775] starting run threads in backend
2024-10-03 18:05:25,547 INFO    MainThread:3252 [wandb_run.py:_console_start():2114] atexit reg
2024-10-03 18:05:25,548 INFO    MainThread:3252 [wandb_run.py:_redirect():1969] redirect: SettingsConsole.WRAP_RAW
2024-10-03 18:05:25,548 INFO    MainThread:3252 [wandb_run.py:_redirect():2034] Wrapping output streams.
2024-10-03 18:05:25,549 INFO    MainThread:3252 [wandb_run.py:_redirect():2059] Redirects installed.
2024-10-03 18:05:25,550 INFO    MainThread:3252 [wandb_init.py:init():817] run started, returning control to user process
2024-10-03 18:05:25,552 INFO    MainThread:3252 [wandb_run.py:_config_callback():1250] config_cb None None {'alias': '', 'experiment_id': '', 'run_tags': [], 'run_notes': '', 'seed': 0, 'device': 'cpu', 'dataset': 'Jet_Engine', 'reciprocal': True, 'model': 'RESCAL', 'rank': 200, 'rank_r': 100, 'init': 0.001, 'dropout': 0, 'lmbda': 0.05, 'world': 'LCWA', 'num_neg': 0, 'score_rel': True, 'score_lhs': False, 'score_rhs': True, 'w_rel': 4.0, 'w_lhs': 1, 'regularizer': 'N3', 'optimizer': 'Adagrad', 'max_epochs': 100, 'valid': 3, 'batch_size': 200, 'learning_rate': 0.1, 'decay1': 0.9, 'decay2': 0.999, 'cache_eval': None, 'model_cache_path': None, 'size': [486, 124, 486], 'loss': 'CrossEntropyLoss()'}
2024-10-03 18:05:25,553 INFO    MainThread:3252 [wandb_watch.py:watch():51] Watching
2024-10-03 18:05:25,565 INFO    MainThread:3252 [jupyter.py:save_ipynb():379] not saving jupyter notebook
2024-10-03 18:05:25,566 INFO    MainThread:3252 [wandb_init.py:_pause_backend():403] pausing backend
2024-10-03 23:09:34,534 INFO    MainThread:3252 [wandb_init.py:_resume_backend():408] resuming backend
2024-10-03 23:09:34,536 INFO    MainThread:3252 [jupyter.py:save_ipynb():379] not saving jupyter notebook
2024-10-03 23:09:34,536 INFO    MainThread:3252 [wandb_init.py:_pause_backend():403] pausing backend
2024-10-03 23:09:41,109 INFO    MainThread:3252 [wandb_init.py:_resume_backend():408] resuming backend
2024-10-03 23:09:41,197 INFO    MainThread:3252 [wandb_setup.py:_flush():68] Configure stats pid to 3252
2024-10-03 23:09:41,197 INFO    MainThread:3252 [wandb_setup.py:_flush():68] Loading settings from C:\Users\shanthod\.config\wandb\settings
2024-10-03 23:09:41,197 INFO    MainThread:3252 [wandb_setup.py:_flush():68] Loading settings from C:\Users\shanthod\OneDrive - Singapore Management University\Desktop\codes\wandb\settings
2024-10-03 23:09:41,197 INFO    MainThread:3252 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True'}
2024-10-03 23:09:41,197 INFO    MainThread:3252 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program_relpath': 'main.py', 'program': 'C:\\Users\\shanthod\\OneDrive - Singapore Management University\\Desktop\\codes\\main.py'}
2024-10-03 23:09:41,197 INFO    MainThread:3252 [wandb_init.py:_log_setup():492] Logging user logs to C:\Users\shanthod\OneDrive - Singapore Management University\Desktop\codes\wandb\run-20241003_230941-9nv8jmyi\logs\debug.log
2024-10-03 23:09:41,197 INFO    MainThread:3252 [wandb_init.py:_log_setup():493] Logging internal logs to C:\Users\shanthod\OneDrive - Singapore Management University\Desktop\codes\wandb\run-20241003_230941-9nv8jmyi\logs\debug-internal.log
2024-10-03 23:09:41,197 INFO    MainThread:3252 [wandb_init.py:init():532] calling init triggers
2024-10-03 23:09:41,197 INFO    MainThread:3252 [wandb_init.py:init():538] wandb.init called with sweep_config: {}
config: {}
2024-10-03 23:09:41,197 INFO    MainThread:3252 [wandb_init.py:init():554] re-initializing run, found existing run on stack: cdvaxk9z
2024-10-03 23:09:41,197 INFO    MainThread:3252 [wandb_run.py:_finish():1854] finishing run shan_proj/ssl-relation-prediction/cdvaxk9z
2024-10-03 23:09:41,197 INFO    MainThread:3252 [jupyter.py:save_history():449] not saving jupyter history
2024-10-03 23:09:41,197 INFO    MainThread:3252 [jupyter.py:save_ipynb():379] not saving jupyter notebook
2024-10-03 23:09:41,197 INFO    MainThread:3252 [wandb_init.py:_jupyter_teardown():420] cleaning up jupyter logic
2024-10-03 23:09:41,202 INFO    MainThread:3252 [wandb_run.py:_atexit_cleanup():2083] got exitcode: 0
2024-10-03 23:09:41,203 INFO    MainThread:3252 [wandb_run.py:_restore():2066] restore
2024-10-03 23:09:41,203 INFO    MainThread:3252 [wandb_run.py:_restore():2072] restore done
